molo/profiles/templates/profiles/done.html

```diff
@@ -1,20 +1,60 @@
 {% extends "base.html" %}
-{% load i18n %}
+{% load i18n static core_tags wagtailsettings_tags wagtailcore_tags %}
+{% get_settings %}
 
 {% block content %}
-<h4 class="heading heading--subtitle">
-    {% trans "Thank you for joining!" %}
-</h4>
 <div class="profiles">
-  <p class="profiles__description">{% trans "Let us know your birthday to get access to exclusive content." %}</p>
-  <form method="post" action="{% url "molo.profiles:registration_done" %}" class="profiles__form">
-  {% csrf_token %}
-    <fieldset>
-      <label for="id_date_of_birth_year">Select date of birth</label>
-      {{ form.date_of_birth }}
-    </fieldset>
+  <h4 class="heading heading--subtitle">
+    {% trans "Thank you for joining!" %}
+  </h4>
+{% with profile_settings=settings.profiles.UserProfilesSettings %}
+
+  {% if profile_settings.activate_dob and not profile_settings.capture_dob_on_reg or profile_settings.activate_display_name and not profile_settings.capture_display_name_on_reg or profile_settings.activate_gender and not profile_settings.capture_gender_on_reg or profile_settings.activate_location and not profile_settings.capture_location_on_reg or profile_settings.activate_education_level and not profile_settings.capture_education_level_on_reg %}
+    <p class="profiles__description">{% trans "Let us know more about yourself to get access to exclusive content." %}</p>
+    <form method="post" action="{% url "molo.profiles:registration_done" %}" class="profiles__form">
+    {% csrf_token %}
+      {% if profile_settings.activate_dob and not profile_settings.capture_dob_on_reg %}
+      <fieldset>
+        <label for="date_of_birth">{% trans "Select Date Of Birth" %}</label><br>
+        {{ form.date_of_birth.errors}}
+        {{ form.date_of_birth }}
+      </fieldset>
+      {% endif %}
+
+      {% if profile_settings.activate_display_name and not profile_settings.capture_display_name_on_reg %}
+      <fieldset>
+        <label for="alias">{% trans "Display Name" %}</label><br>
+          {{ form.alias.errors }}
+          {{ form.alias }}
+      </fieldset>
+      {% endif %}
+
+      {% if profile_settings.activate_gender and not profile_settings.capture_gender_on_reg %}
+      <fieldset>
+        <label for="gender">{% trans "Gender" %}</label><br>
+        {{ form.gender.errors }}
+        {{ form.gender }}
+      </fieldset>
+      {% endif %}
+
+      {% if profile_settings.activate_location and not profile_settings.capture_location_on_reg %}
+      <fieldset>
+        <label for="location">{% trans "Location" %}</label><br>
+        <input class="form-control" {{ form.location.errors }}{{ form.location }}<br>
```

Comment:
Please take out the `{{ form.location.errors }} and {{ form.location }}` from the input tag as well